snippets/elixir.snippets

```diff
@@ -1,81 +1,81 @@
 snippet do
 	do
-		${0}
+		${0:expression}
```

Comment:
It's good in some places, but when everything in a language is an expression, saying that something's an expression doesn't mean much :)